src/System Application/App/Email/src/Message/EmailAttachments.Page.al

```diff
@@ -53,15 +54,41 @@ page 8889 "Email Attachments"
     {
         area(Processing)
         {
+            fileuploadaction(UploadMultiple)
+            {
+                ApplicationArea = All;
+                Image = Attach;
+                Caption = 'Add files';
+                ToolTip = 'Attach files, such as documents or images, to the email.';
+                Scope = Page;
+                Visible = IsEmailEditable;
 
+                trigger OnAction(files: List of [FileUpload])
+                var
+                    EmailEditor: Codeunit "Email Editor";
+                    TempInStream: InStream;
+                    SingleFile: FileUpload;
+                begin
+                    foreach SingleFile in files do begin
```

Comment:
This should be moved to the codeunit, would make sense partners can also call EmailEditor.UploadAttachment(files)

Reply:
Unfortunately now we have bug: FileUpload cannot be used in Codeunit. Will refactor this to a codeunit once this bug is solved